Library/Formula/mplayer.rb

```diff
@@ -22,47 +24,16 @@ class Mplayer < Formula
     patch :DATA
   end
 
-  option "without-osd", "Build without OSD"
-
   depends_on "yasm" => :build
   depends_on "libcaca" => :optional
-  depends_on :x11 => :optional
-
-  deprecated_option "with-x" => "with-x11"
-
-  if build.with?("osd") || build.with?("x11")
-    # These are required for the OSD. We can get them from X11, or we can
-    # build our own.
-    depends_on "fontconfig"
-    depends_on "freetype"
-    depends_on "libpng"
-  end
-
-  fails_with :clang do
-    build 211
-    cause "Inline asm errors during compile on 32bit Snow Leopard."
-  end unless MacOS.prefer_64_bit?
-
-  # ld fails with: Unknown instruction for architecture x86_64
-  fails_with :llvm
 
   def install
-    # It turns out that ENV.O1 fixes link errors with llvm.
-    ENV.O1 if ENV.compiler == :llvm
```

Comment:
does this mean both the fixed --cc= argument as well as the --host-cc= argument should stay in or just the --cc= (in args = %W[...)